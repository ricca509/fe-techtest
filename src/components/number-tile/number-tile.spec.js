import NumberTile from ".";

describe("NumberTile", () => {
  it("should render a box containing a number", () => {
    const result = NumberTile({ ordinal: 5 });

    expect(result).toMatch("5");
  });

  it("should render a list item and have the current markup", () => {
    const result = NumberTile({ ordinal: 5 });

    expect(result).toMatchSnapshot();
  });
});
