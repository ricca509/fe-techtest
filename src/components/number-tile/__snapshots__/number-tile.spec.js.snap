// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberTile should add the right class and data attribute when selected 1`] = `
"    
    <button data-role=\\"number\\" data-number=\\"5\\" data-selected=\\"true\\" class=\\"number selected\\">
      5
    </button>  
  "
`;

exports[`NumberTile should render a list item and have the correct markup 1`] = `
"    
    <button data-role=\\"number\\" data-number=\\"5\\" data-selected=\\"false\\" class=\\"number\\">
      5
    </button>  
  "
`;
